apps/examples/mediaplayer/MediaPlayerAppWifi.cpp

```diff
@@ -0,0 +1,104 @@
+/****************************************************************************
```

Comment:
This is kind of utils for media player app, also already in mediaplayer. How about connectivityWifi.cpp or something?

Reply:
I think WifiConnector is better.
Since WifiConnector is common name, I'll add MediaPlayerApp namespace.
So new class will be MediaPlayerApp::WifiConnector.